k8s/helm/yona/templates/03_job-liquibase.yaml

```diff
@@ -12,8 +12,37 @@ metadata:
 spec:
   template:
     metadata:
+      labels:
+        app: liquibase
+        build: {{ .Chart.AppVersion | quote }}
+        stage: {{ .Values.global.stage | default "develop" }}
       name: {{ .Values.global.stage | default "develop" }}-yona-liquibase-update
     spec:
+      initContainers:
+        - name: validatedb
+          image: craftypenguins/k8s-init-mysql:latest
+          env:
+            - name: DBUSER
+              value: {{ .Values.mariadb.mariadbUser | default "develop" | quote }}
+            - name: DBPASSWORD
+              value: {{ .Values.mariadb.mariadbPassword | default "develop" | quote }}
+            - name: DBHOST
+              value: {{ .Values.mariadb.mariadbHostname | quote }}
+            - name: DBNAME
+              value: {{ .Values.mariadb.mariadbDatabase | quote }}
+            - name: TIMEOUT
+              value: "10s"
+        - name: validateldap
```

Comment:
It's a bit strange to validate LDAP here, as this job has no LDAP dependency. Only the app service depends on LDAP.

Reply:
The liquibase container also populates the ldap at bootstrap so it is a dependency.